include/targets/generic/block_reduction_kernel_host.h

```diff
@@ -5,6 +5,7 @@ namespace quda
   {
     Functor<Arg> t(arg);
     dim3 block(0, 0, 0);
+#pragma omp parallel for
```

Comment:
This will only parallelize the outer loop. If you want to parallelize both loops add the `collapse(2)` clause e.g.:
```
#pragma omp parallel for collapse(2)
for (block.y = 0; ... ) {
   for( block.x = 0; ...) {
    ...
    }
 }
 ```

Reply:
Marking as resolved: see conversation [above](https://github.com/lattice/quda/pull/1479#discussion_r1671042901).